### PR TITLE
preR13 fixes

### PR DESCRIPTION
**Pull request by michal-josef-spacek**

### CODE REVIEW

**Comment by rurban** [comment]
Oops, missed that, sorry.

File: include/dwg.h [hunk 1]
```diff
@@ -2339,6 +2339,7 @@ typedef struct _dwg_object_BLOCK_HEADER
   BITCODE_H layout;
   BITCODE_RS unknown_r11;
   BITCODE_RS unknown1_r11;
+  BITCODE_RC r11_unknown;
```
Comment (rurban) [inline include/dwg.h:2342]: Rather re-use the existing RS with FIELD_CAST (unknown_r11, RC, RS, 0); 
There's no overlap, and maybe they do mean the same

File: src/decode_r11.c [hunk 2]
```diff
@@ -328,6 +328,8 @@ decode_preR13_section (Dwg_Section_Type_r11 id, Bit_Chain *restrict dat,
             PRE (R_11) {
               FIELD_CAST (num_owned, RS, BL, 0);
               FIELD_RC (flag2, 0);
+              if (dwg->header.numheader_vars == 74)
+                FIELD_RC (r11_unknown, 0);
```
Comment (rurban) [inline src/decode_r11.c:332]: FIELD_CAST (unknown_r11, RC, RS, 0);

File: src/decode_r11.c [hunk 3]
```diff
@@ -348,6 +350,8 @@ decode_preR13_section (Dwg_Section_Type_r11 id, Bit_Chain *restrict dat,
             FIELD_CMC (color, 62); // off if negative
             PRE (R_11) {
               FIELD_HANDLE (ltype, 2, 6);
+              if (dwg->header.numheader_vars == 74)
+                FIELD_RC (r11_unknown, 0);
```
Comment (rurban) [inline src/decode_r11.c:354]: unknown_r2 instead

File: src/decode_r11.c [hunk 4]
```diff
@@ -430,6 +434,10 @@ decode_preR13_section (Dwg_Section_Type_r11 id, Bit_Chain *restrict dat,
               FIELD_3RD (VIEWDIR, 11);
             if (tbl->size > 89)
               FIELD_RS (flag_3d, 0);
+            PRE (R_10) {
+              if (dwg->header.numheader_vars == 74)
+                FIELD_RC (r11_unknown, 0);
```
Comment (rurban) [inline src/decode_r11.c:439]: unknown_r2 instead

File: include/dwg.h [hunk 5]
```diff
@@ -2512,6 +2514,7 @@ typedef struct _dwg_object_VIEW
   BITCODE_H named_ucs;
   BITCODE_H livesection;
   BITCODE_RS flag_3d;
+  BITCODE_RC r11_unknown;
```
Comment (rurban) [inline include/dwg.h:2517]: unknown_r2 instead

File: include/dwg.h [hunk 6]
```diff
@@ -2374,6 +2375,7 @@ typedef struct _dwg_object_LAYER
   BITCODE_H material;     /* DXF 347 */
   BITCODE_H ltype;        /* DXF 6 */
   BITCODE_H visualstyle;  /* DXF 348 */
+  BITCODE_RC r11_unknown;
```
Comment (rurban) [inline include/dwg.h:2378]: unknown_r2 instead

**Review by rurban — APPROVED** [review]
I merged it, and fixed the unknowns by myself. Thanks

**Comment by michal-josef-spacek** [comment]
@rurban thanks

**Pull request closed without merging**